### ISSUE TITLE
Pre-trained model 

### ISSUE THREAD

**Issue by zjxyz** [yhhhli/BRECQ#26]
Hi,
Very impressive coding.

https://github.com/yhhhli/BRECQ/releases/download/v1.0/resnet50_imagenet.pth.tar 
I tested the pre-trained model resnet50 with only 76.62 performance, but the paper wrote 77.00.

Thanks

**Comment by yhhhli** [comment]
Can you show your code of evaluation?

**Comment by zjxyz** [comment]
> Can you show your code of evaluation?
It is tested with the code of brecq.
In  ( main_imagenet.py ##199 ), I add code:
    qnn.set_quant_state(weight_quant=False, act_quant=False)
    print('accuracy: {}'.format(validate_model(test_loader, qnn)))
Tested the pre-trained model resnet50 with only 76.62 performance.
